src/components/Settings/RowOne.tsx

```diff
@@ -190,22 +190,28 @@ export default function RowOne({ t }: { t: Locale }) {
 
       <div className="w-full flex flex-col lg:flex-row gap-8 mt-8 mb-20 lg:px-24 items-center">
         <div className="flex flex-col max-sm:gap-y-8 sm:flex-row sm:gap-x-8">
-          <UploadButton
-            t={t}
-            selectedImage={whiteLogo}
-            setSelectedImage={setWhiteLogo}
-            textAbove={"Vit Logga"}
-            textInside={"Logga"}
-            accept={"image/svg+xml"}
-          />
-          <UploadButton
-            t={t}
-            selectedImage={colorLogo}
-            setSelectedImage={setColorLogo}
-            textAbove={"Logga m. färg"}
-            textInside={"Logga"}
-            accept={"image/png,image/jpeg"}
-          />
+          <div>
+            <UploadButton
+              t={t}
+              selectedImage={whiteLogo}
+              setSelectedImage={setWhiteLogo}
+              textAbove={"Vit Logga"}
+              textInside={"Logga"}
+              accept={["image/svg+xml"]}
+            />
+            <p className="text-neutral-300">{t.exhibitorSettings.table.row1.section1.only} svg</p>
```

Comment:
I think these can be integrated into the design in a concise way but we can always change it later.

Reply:
I agree